### ISSUE TITLE
Web Portal suggestion

### ISSUE THREAD

**Issue by Hrovatin** [theislab/sfaira-portal#2]
Maybe we could collect what else we would like to have on the web portal or what could be improved. @lazappi feel free to close if not relevant.

Here are some ideas:
- Maybe better to show by study and then expanding for sample as else it is hard to look at.
- When panels are expanded maybe sort Cell types filed somehow (e.g. by name or by number of cells).
- Searching by GEO/doi so that it is easier to figure out if something still needs to be added.

**Comment by Hrovatin** [comment]
Any updates on this? I think GEO ID search would be really useful.

**Comment by davidsebfischer** [comment]
Yes it will be! We havent managed to update the website yet though.
